src/graph/graph.c

```diff
@@ -487,7 +399,7 @@ int Graph_AddLabelMatrix(Graph *g) {
         g->_labels = realloc(g->_labels, g->label_cap * sizeof(GrB_Matrix));
     }
 
-    GrB_Matrix_new(&g->_labels[g->label_count++], GrB_BOOL, g->node_cap, g->node_cap);
+    GrB_Matrix_new(&g->_labels[g->label_count++], GrB_BOOL, _Graph_NodeCap(g), _Graph_NodeCap(g));
```

Comment:
`_Graph_Resize_Matrix` will shrink matrices to NodeCount before they get populated, so that would probably be a better allocation size.

Reply:
It might be that we create a label matrix before calling `Graph_CreateNodes`, in which case, I think  it's better creating a matrix with dimensions `_Graph_NodeCap(g) X _Graph_NodeCap(g)`